src/frontends/tensorflow/src/utils.hpp

```diff
@@ -175,8 +175,9 @@ void values_from_const_node(const NodeContext& node, ov::Shape* const_tensor_sha
                 FRONT_END_THROW("Encountered unknown element type " + DataType_Name(dt) + " on an empty tensor_proto");
             }
             TENSORFLOW_OP_VALIDATION(node, val_size != 0, "Empty values vector");
```

Comment:
Should we delete this exception? It looks like it will be thrown first before execution lines below

Reply:
Yes, this line must be deleted